### ISSUE TITLE
switch_case_on_newline #selector false positive

### ISSUE THREAD

**Issue by mminer** [realm/SwiftLint#1020]
The `switch_case_on_newline` rule incorrectly detects a violation when ones of the cases is a #selector that includes a colon, e.g. in this function:

```swift
override func validateMenuItem(_ menuItem: NSMenuItem) -> Bool {
    let selector = menuItem.action!

    switch selector {
    case #selector(aFunction(_:)):
        return false
    default:
        return true
    }
}
```

**Comment by marcelofabri** [comment]
I had no idea you could `switch` a selector 😳

**Comment by marcelofabri** [comment]
@mminer Can you try #1023 to see if it solves the issue for you?

**Comment by mminer** [comment]
@marcelofabri Works well for me; all the false positives in my project are gone. Thanks for the (incredibly fast) fix.
